### PR TITLE
storage: upgrade MinIO to its latest version

### PR DESCRIPTION
**Pull request by humitos**
Following #89 to upgrade MinIO to its latest version.

Blocked because it requires updating the documentation from https://docs.readthedocs.io/en/latest/development/install.html with the new steps required since the MinIO Console has changed its look&feel.

### CODE REVIEW

**Review by stsewd — APPROVED** [review]

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Can we have docs? 🙃 I can write them myself by pushing to this branch before merging. As far as I understand there's no rush to upgrade MinIO?

**Comment by humitos** [comment]
@astrojuanlu we can! I didn't request a review on this one and marked it as blocked because the MinIO UI has changed considerably and I didn't want to jump into it to learn how to do what we are doing in the new UI.

Note the documentation we have to update is on `readthedocs.org` repository at https://docs.readthedocs.io/en/latest/development/install.html

As you already guessed, there is no rush on merging this PR. I just wanted to open it with the new configs because I've already have read about them.

**Comment by humitos** [comment]
One extra note: we may need to read if there is any step required to "migrate the data". I don't know if the data structure has changed in this new release, but is worth taking a look at that as well.

**Comment by astrojuanlu** [comment]
> One extra note: we may need to read if there is any step required to "migrate the data".

How relevant is this for development installations? I mean: sure, it's a pain in the neck having to recreate some projects. I'm just weighing how much effort do we want to put in documenting that.

**Comment by humitos** [comment]
Yeah, I'm fine dropping the storage and re-creating it honestly. I don't think we need to document it for users and make it public in our docs; I was thinking just about _knowing_ if it's required and if there is something like "execute _this_ command once, and it's done" for the core team only. I'm fine having _that_ command in the description of the PR only.

**Comment by humitos** [comment]
I just checked this locally and there is no need to migrate data.

**Pull request merged**
